source/connect.txt

```diff
@@ -123,3 +141,88 @@ For details, see :ref:`comp-command-line-connect`.
    Favorite Connections </connect/favorite-connections>
    Required Access </connect/required-access>
    Disconnect </connect/disconnect>
+
+Connection String Examples
+--------------------------
+
+The following examples highlight some commonly misunderstand advanced connection
+string options, particularly when connecting to Compass. For information on 
+all connection string options, see :ref:`connections-connection-options`.  
+
+TLS Options
+~~~~~~~~~~~
+
+The following connection string enables TLS using the ``tls`` URI option. When
+specifying TLS/SSL certificates in your connection string, you must specify the ``tlsCAFile``
+as the path to the Certificate Authority file, URL-encoded, and the ``tlsCertificateKeyFile``
+as the path to the combined server certificate and private key file, URL-encoded.
```

Comment:
No – `tlsCertificateKeyFile` is the path to the _client_ certificate and private key and is often optional, unless the server is set up to require client certificates or the user tries to use X.509 auth